advanced-usage/index.md

```diff
@@ -83,6 +119,8 @@ Multiple simultaneous tabs from the same browser instance will be supported in f
 
 - Open edit mode via the [burger menu](#burger-menu)
 - Select a view to edit, and/or create or remove views as desired
+    - Clicking on the cog settings icon allows renaming a view, and determining whether the bottom
```

Comment:
"Even if you choose to have it hidden on boot, you are able to show and hide it during operation with the TOGGLE_BOTTOM_BAR action."

Reply:
Actions will be documented later because they're still actively being changed.